### ISSUE TITLE
New release (0.10?)

### ISSUE THREAD

**Issue by phadej** [haskell-servant/servant#655]
There haven't been any releases in two month, though stuff got merged in.

Should we cut a release, or is there some blocking issues?

**Comment by alpmestan** [comment]
@phadej I definitely think we can (and should) cut a release, it only gets "heavier" with time so it would be nice to have one out soonish. This would have the nice side effect of fixing the servant-related breakage in Stackage -- see https://github.com/fpco/stackage/issues/2177.

**Comment by phadej** [comment]
If there's anything I can help with, please tell.

**Comment by alpmestan** [comment]
Have you seen my email from early this week?

**Comment by phadej** [comment]
> New servant maintainers

Yes I have, and I'm sure I replied that's I'll can be co-maintainer, but my email client says I didn't send the reply.

**Comment by alpmestan** [comment]
You did not indeed =) Let's discuss that by email, the 0.10 release should be one of the first things we schedule.

**Comment by phadej** [comment]
@alpmestan  I send a new email (right after your comment), have you got it? or is there something flaky with my client

**Comment by TomMD** [comment]
A new release is even more important now that the Aeson dep has been bumped.  Thoughts?

**Comment by phadej** [comment]
See https://github.com/haskell-servant/servant/milestone/3

We are almost there. Hopefully will be able to cut the release next week

**Comment by TomMD** [comment]
@phadej Would you be willing to make a point release that just eases the bounds?

**Comment by phadej** [comment]
@TomMD relaxed bounds for `servant`, `servant-client` and `servant-server`

**Comment by phadej** [comment]
Would like not to propagate them more, as major release will require very similar work next week anyway. Based on stackage aeson-1.1 issue, there is no hurry.

**Comment by phadej** [comment]
Blocked by #692

**Comment by axman6** [comment]
So, this happened, can this be closed? =)

**Comment by phadej** [comment]
thanks for the reminder
